packages/test/local-server-tests/src/test/connectionMode.spec.ts

```diff
@@ -20,11 +20,11 @@ import {
 } from "@fluidframework/server-local-server";
 import { MockLogger } from "@fluidframework/telemetry-utils/internal";
 import {
+	createAndAttachContainerUsingProps,
```

Comment:
Lingering from previous refactoring. But as this corrects the alphabetical ordering, may as well keep it.